From 5fab2468f55d4209edf98bb3b00b5d3ec5b6d909 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Fri, 13 Jul 2018 10:52:33 -0500
Subject: [PATCH] Add dependency for restoring packages before trying to find
 Microsoft.NETCore.DotNetAppHost package.

This appears to be a difference between the 2.2.0-preview SDK currently
being used in core-sdk and the 2.1.401-preview SDK currently being used
in source-build.  In the 2.2.0 SDK, this target happens to run before
the resolved package list is used, but in the 2.1.401 it is not, so I
added the explicit dependency (doesn't affect anything in the 2.2.0 SDK).

This patch can be removed after https://github.com/dotnet/core-sdk/pull/34
is merged.
---
 src/redist/redist.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
index ac8ccb4de..73597e874 100644
--- a/src/redist/redist.csproj
+++ b/src/redist/redist.csproj
@@ -77,7 +77,7 @@
     </MSBuild>
   </Target>
 
-  <Target Name="PublishAppHostTemplate">
+  <Target Name="PublishAppHostTemplate" DependsOnTargets="RunResolvePackageDependencies">
 
     <PropertyGroup>
       <NETCoreDotNetAppHostPackageName>Microsoft.NETCore.DotNetAppHost</NETCoreDotNetAppHostPackageName>
-- 
2.14.1

